### ISSUE TITLE
[BUG] Swiper WCAG issue

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#902]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I tab to the swiper and get focus, there is no tooltip and using the arrow doesn't work. If I tab again, the focus is on the map and arrows move the map

### Expected Behavior

There should be only one focus that shows tooltip and move the swiper with the arrows

### Steps To Reproduce

1. Go to package-swiper.html
2. Tab to the swiper of map 2 (on focus arrows not working)
3. Tab again, focus on the map and arrows move the map

**Comment by elatsis** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/GeoView-Web-Team-6332f4560a41153d1fefbcbc/issues/Canadian-Geospatial-Platform/geoview/902?planning-poker) with Zenhub @kaminderpal
